### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by xuefer** [microg/android_packages_apps_UnifiedNlp#66]
```
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.os.Bundle.containsKey(java.lang.String)' on a null object reference
    at org.microg.tools.selfcheck.NlpStatusChecks$2.onLocationChanged(NlpStatusChecks.java:106)
    at android.location.LocationManager$ListenerTransport._handleMessage(LocationManager.java:285)
    at android.location.LocationManager$ListenerTransport.-wrap0(LocationManager.java)
    at android.location.LocationManager$ListenerTransport$1.handleMessage(LocationManager.java:230)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:148)
    at android.app.ActivityThread.main(ActivityThread.java:5466)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
    at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:117)
```

CM-13.0. UnifiedNlp.apk is moved to /system/priv-app/UnifiedNlp/UnifiedNlp.apk, resource override'd using xposed
exception thrown after clicking "Self-check"

using UnifiedNlp.apk 1.6.0 (but apps says 1.5.1)

**Comment by mar-v-in** [comment]
Should be fixed in v1.6.1

**Comment by xuefer** [comment]
```
03-04 23:44:07.035 14145 14145 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.os.Bundle.containsKey(java.lang.String)' on a null object reference
03-04 23:44:07.035 14145 14145 E AndroidRuntime:        at org.microg.tools.selfcheck.NlpStatusChecks$2.onLocationChanged(NlpStatusChecks.java:106)
03-04 23:44:07.035 14145 14145 E AndroidRuntime:        at android.location.LocationManager$ListenerTransport._handleMessage(LocationManager.java:285)
03-04 23:44:07.035 14145 14145 E AndroidRuntime:        at android.location.LocationManager$ListenerTransport.-wrap0(LocationManager.java)
03-04 23:44:07.035 14145 14145 E AndroidRuntime:        at android.location.LocationManager$ListenerTransport$1.handleMessage(LocationManager.java:230)
03-04 23:44:07.035 14145 14145 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:102)
03-04 23:44:07.035 14145 14145 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:148)
03-04 23:44:07.035 14145 14145 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:5466)
03-04 23:44:07.035 14145 14145 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
03-04 23:44:07.035 14145 14145 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
03-04 23:44:07.035 14145 14145 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```
reproduced in 1.6.1 too

from the exception, it's containsKey's "this" is null, which mean location.getExtras() is null, not location
